proto/interchain_security/ccv/consumer/v1/query.proto

```diff
@@ -52,3 +56,17 @@ message QueryParamsResponse {
   // params holds all the parameters of this module.
   Params params = 1 [ (gogoproto.nullable) = false ];
 }
+
+message QueryProviderInfoRequest {}
+
+message QueryProviderInfoResponse {
+  ChainInfo consumer = 1 [ (gogoproto.nullable) = false ];
+  ChainInfo provider = 2 [ (gogoproto.nullable) = false ];
+}
+
+message ChainInfo {
+  string ChainID = 1;
```

Comment:
nit: Do the `ChainId`, `ClientId`, etc. have to have a capital first letter? If not, would it make sense to have a lower case first letter to be consistent with other `message`s (e.g., `QueryProviderInfoResponse` where the field `consumer` is not capitalized).

Reply:
Hi, i checked other proto files, the style is `chain_id`